SampleFunctionApp/local.settings.json.Sample.json

```diff
@@ -17,18 +6,18 @@
 
     // OidcApiAuthorizationSettings:Audience setting.
     // Identifies the API to be authorized by the Open ID Connect provider (issuer).
-    // The "Audiance" is the indentifer used by the authorization provider to indentify
+    // The "Audience" is the indentifer used by the authorization provider to indentify
     // the API (HTTP triggered Azure Function) being protected. This is often a URL but
     // For Auth0 the Audience setting set here should match the API's Identifier
     // in the Auth0 Dashboard.
-    "OidcApiAuthorizationSettings:Audience": "Your-API-Identifier-goes-here",
+    "OidcApiAuthorizationSettings:Audience": "sVS4mpntKsbjo54g3jirW8Qfd6gcfuLl",
```

Comment:
Don't commit you're audience to source control. Leave the value set to `"Your-API-Identifier-goes-here"`. :-)